tests/integration/security/filebased_tls_origination/destination_rule_tls_test.go

```diff
@@ -41,9 +43,9 @@ func mustReadFile(t *testing.T, f string) string {
 	return string(b)
 }
 
-// TestDestinationRuleTls tests that MUTUAL tls mode is respected in DestinationRule.
+// TestDestinationRuleTLS tests that MUTUAL tls mode is respected in DestinationRule.
```

Comment:
nit: I had recently changed this to TestDestinationRuleTls to be more consistent with the way we write Tls in our code, while I agree TLS is more apt but can you change it back Tls so that we are not inconsistent, maybe later down the line we change all Tls to TLS